index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/styles.css" />
+  <title>My Portifolio</title>
+</head>
+
+<body>
+  <nav class="flexr container">
+    <a href="#" class="link">
+      <img src="https://arsenethierry-dev.vercel.app/images/logo-light.svg" width="90px" height="60px" alt="github icon" />
+    </a>
+    <img src="/images/menu button.png" width="18px" height="12px" id="menu" alt="menu icon" />
+    <ul>
+      <li><a href="about.html" class="link">Portfolio</a></li>
+      <li><a href="About.html" class="link">About</a></li>
+      <li><a href="contact.html" class="link">Contact</a></li>
+    </ul>
+  </nav>
+  <section class="container flexc" id="header">
+    <section class="flexc">
+      <h1>I’m Arsene Thierry. Glad to see you!</h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hesitate to contact
+        me.
+      </p>
+      <h3><a href="#" class="link">LET’S CONNECT</a></h3>
+      <div class="flexr">
+        <ul>
+          <li>
+            <a href="#" class="link">
+              <img src="/images/twitter.png" width="20px" height="20px" alt="gmail icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#" class="link">
+              <img src="/images/gmail.svg" width="20px" height="20px" alt="gmail icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#" class="link">
+              <img src="/images/Linkedin icon.png" width="20px" height="20px" alt="Linkedin icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#" class="link">
+              <img src="/images/Vector.svg" width="20px" height="20px" alt="github icon" />
+            </a>
+          </li>
+          <li>
+            <a href="#" class="link">
+              <img src="/images/Vector peace.svg" width="20px" height="20px" alt="peace icon" />
+            </a>
+          </li>
+        </ul>
```

Comment:
- [ ] Please fix this problem with the rendering the icons on the mobile. I can see them on desktop version, but they are lost in mobile. 
![image](https://user-images.githubusercontent.com/91486791/235221795-eea58d34-47bc-446b-a5f2-858ecfaf9b00.png)
desktop version: 
![image](https://user-images.githubusercontent.com/91486791/235221996-0fa12cb2-bd0c-4ad0-999e-72ec45e40438.png)